### DISCUSSION TITLE
date formatting

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3560] [General]
Has anyone formatted the date using moment.js? I'm trying to add days to the current date.

**Comment by twilson9034** [comment]
I've tried using moment.js to add the days and done everything i can think of and just cant get it to work.

I can format the date itself i.e. {{date now "dddd" }} will give just the day of the current date but if I try to do anything other than just that I get invalid binding, invalid date or incorrect syntax.

**Reply by chadgriff** [reply]
Check out issue #1910 which has a formula to subtract days.  May be relevant to your add days use case.

**Reply by twilson9034** [reply]
That issue helped, Thank you @chadgriff. 
@mjashanks is it in the works to have this type of info in the docs?
